source/gui/exit.py

```diff
@@ -0,0 +1,163 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2006-2022 NV Access Limited, Peter Vágner, Aleksey Sadovoy, Mesar Hameed, Joseph Lee,
+# Thomas Stivers, Babbage B.V., Accessolutions, Julien Cochuyt, Cyrille Bougot
```

Comment:
Did you confirm this history for this class?

Reply:
This was just copied from `gui/__init__.py`, I think it is much safer and easier to preserve all names when splitting up files.